docs/developers/customization.rst

```diff
@@ -111,16 +111,44 @@ On the contrary, if you want to reference images from HTML code or inline CSS,
 you should use the ``{% static %}`` template tag.
 
 
-.. _customization#building:
+.. _customization#install-node-libs:
+
+Installing JS build libraries
+-----------------------------
+
+Before you can rebuild your rebuild your static assets with any CSS or
+Javascript customisations, you will need to install some node libraries:
+
+Before proceeding please make sure you have
+:ref:`node and npm installed in your system <requirements#customize-static>`.
+
+.. code-block:: bash
+
+   (env) $ cd $pootle_dir/pootle/static/js/
+   (env) $ npm install
+
+``$pootle_dir`` is the directory where Pootle is installed.
```

Comment:
how about:
```
(env) $ cd $pootle_dir
(env) $ cd pootle/static/js
```
On this point ive been thinking about implementing a ``pootle --info`` command or similar - which would tell you where all your important directories and eg settings file were.